### PR TITLE
feat(docker): docker files for running the server

### PR DESCRIPTION
**Pull request by lxndr**

### CODE REVIEW

**Comment by kuzn-ilya** [comment]
Maybe it's worth not to put docker files to sub folder?

File: docker/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      context: .
+      dockerfile: backend.dockerfile
+    depends_on:
+      - database
+    environment:
+      DATABASE_HOST: database
+    ports:
+      - 1337:1337
+
+volumes:
```
Comment (eugrdn) [inline docker/docker-compose.yaml]: is it used?

File: docker/backend.dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM node:10-alpine AS build
+RUN apk add git automake autoconf build-base zlib-dev
+
+USER node
+WORKDIR /home/node
+RUN git clone --depth=1 https://github.com/kuzn-ilya/ryazan.git
+
+WORKDIR /home/node/ryazan/server
+RUN sed -i 's/127.0.0.1/database/g' "./config/environments/development/database.json"
```
Comment (eugrdn) [inline docker/backend.dockerfile]: can't see this config anywhere 😅

File: docker/docker-compose.yaml (hunk 1 above)
Comment (lxndr) [reply docker/docker-compose.yaml]: volumes? yes, pg and pgadmin need persistent data

**Comment by lxndr** [comment]
Any suggestions about the proper place for the folder?

File: docker/backend.dockerfile (hunk 2 above)
Comment (lxndr) [reply docker/backend.dockerfile]: It's a strapi config. and this is a workaround. strapi development config can only connect to `localhost` but `database` host is needed. I didn't want to change anything.

**Comment by lxndr** [comment]
Considering that I plan to add .env file here, I'd keep this in one folder but rather moved it somewhere.

**Comment by eugrdn** [comment]
I think it's ok and common case to keep docker stuff in the root

**Pull request closed without merging**